cypress/integration/a11y/pni_assessments.js

```diff
@@ -0,0 +1,68 @@
+import { viewports, A11Y_CONFIG, EXCLUDE_PNI_CONSTANTS } from './global_variables'
+
+let dimension;
+
+const PNI_A11Y = () => {
+    viewports.forEach((viewport) => {
+        // FOMO
```

Comment:
this comment can probably be removed?

Reply:
rather, renamed. These are the homepage tests.

Reply:
you're right, not much point having this here.